src/pages/home/report/ReportActionItem.js

```diff
@@ -95,8 +98,18 @@ class ReportActionItem extends Component {
                             )}
                             <View style={getReportActionItemStyle(hovered)}>
                                 {!this.props.displayAsGroup
-                                    ? <ReportActionItemSingle action={this.props.action} />
-                                    : <ReportActionItemGrouped action={this.props.action} />}
+                                    ? (
+                                        <ReportActionItemSingle
+                                            action={this.props.action}
+                                            isOffline={this.props.isOffline}
+                                        />
+                                    )
+                                    : (
+                                        <ReportActionItemGrouped
+                                            action={this.props.action}
+                                            isOffline={this.props.isOffline}
+                                        />
+                                    )}
```

Comment:
Instead of drilling this prop down would it make sense to have `ReportActionItem` subscribe to the network key?

Reply:
Ok Sure.

Reply:
Sorry I misspoke here I meant to say, we should not need to pass `isOffline` prop to both `ReportActionItemSingle` and `ReportActionItemGrouped`. We can subscribe in `ReportActionItemMessage`.

Reply:
Oh.